src/evaluate.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import numpy as np
+import re
+from src.model import yoModel
+from tqdm import tqdm
+from typing import List, Tuple
+from src.utils import WORDS_REGEX
+from dataclasses import dataclass
+from sklearn.metrics import accuracy_score, precision_score, recall_score, \
+    f1_score, roc_auc_score
+
+
+@dataclass
+class EvaluateResult:
+    """Model evaluation result."""
+    accuracy: float
+    precision: float
+    recall: float
+    f1_score: float
+    auroc: float
+
+    def __str__(self):
+        return "\n".join(["{}: {}".format(m, v) for m, v in self.__dict__.items()])
+
+
+def get_substrings(text: str) -> List[List[int]]:
```

Comment:
Вроде как тут другая сигнатура:
```suggestion
def get_substrings(text: str) -> List[Tuple[int]]:
```